docs/references/_attachments/ic.did

```diff
@@ -248,6 +248,16 @@ type canister_status_result = record {
     settings : definite_canister_settings;
     module_hash : opt blob;
     memory_size : nat;
+    memory_metrics : record {
+        heap_size : nat;
```

Comment:
Shouldn't this be called `wasm_memory_size` analogously to, e.g., `wasm_memory_limit`?